packages/worker/src/rpcProvider/webSocketProviderExtended.ts

```diff
@@ -1,73 +1,18 @@
+import { Logger } from "@nestjs/common";
 import { providers } from "ethers";
```

Comment:
I'd rather keep the latest version

Reply:
Do you think it's worth keeping it considering that we need to cover it with tests? I was thinking about removing it in both worker and data-fetcher.